app/policies/placements/user_membership_policy.rb

```diff
@@ -0,0 +1,13 @@
+class Placements::UserMembershipPolicy < ApplicationPolicy
+  def destroy?
+    return false unless user != record.user
```

Comment:
Would this logic be clearer the other way around?

```suggestion
    # Users cannot delete themselves
    return false if user == record.user
```

To me, this feels easier to parse. "Return false if the current user is the `record.user`."